### PR TITLE
FUSETOOLS2-2322: Add UI test for Update exchange variable values whil…

### PR DESCRIPTION
**Pull request by pospisilf**
…e debugging

### CODE REVIEW

**Comment by pospisilf** [comment]
not working with camel 4.4.0 - bug or feature?

**Comment by apupier** [comment]
> not working with camel 4.4.0 - bug or feature?

Camel Exchange variables were introduced in Camel 4.4 https://camel.apache.org/manual/variables.html

**Comment by pospisilf** [comment]
> > not working with camel 4.4.0 - bug or feature?
> 
> Camel Exchange variables were introduced in Camel 4.4 https://camel.apache.org/manual/variables.html

So with Camel 4.4.0 this should be working, right?

**Comment by pospisilf** [comment]
I am getting Internal Error when using 4.4.0. On the other hand 4.4.1 is working properly. 
image

**Comment by pospisilf** [comment]
Required method was missing in 4.4.0, fixed in 4.4.1 (https://issues.apache.org/jira/browse/CAMEL-20445).

File: src/ui-test/tests/variables.test.ts [hunk 1]
```diff
@@ -34,7 +35,7 @@ import {
     isVersionNewer,
     DEBUGGER_ATTACHED_MESSAGE,
 } from '../utils';
-import { RESOURCES_DIR, VARIABLESTEST_JAVA } from '../variables';
+import { RESOURCES_DIR, VARIABLESTEST_YAML } from '../variables';
```
Comment (djelinek) [inline src/ui-test/tests/variables.test.ts:38]: why was the change from` Java DSL` to `YAML DSL` needed?

**Review by djelinek — APPROVED** [review]

File: src/ui-test/tests/variables.test.ts (hunk 1 above)
Comment (pospisilf) [reply src/ui-test/tests/variables.test.ts:38]: I think it's better to have YAML as it's preferred format. There is no difference in functionality.

**Pull request merged**